Profile Folder/chrome/locale/ja/properties/gsettings.properties

```diff
@@ -1,9 +1,9 @@
-addAnApp=Add an app
+addAnApp=アプリを追加する
```

Comment:
Geckium Settings now has a new string here: `geckiumSettings=Geckium Settings`.
Please update this file, thank you :)